src/components/loader-bar/loader-bar.component.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styledSystemPropTypes from "@styled-system/prop-types";
+import tagComponent from "../../utils/helpers/tags";
+import StyledLoaderBar, { InnerBar, StyledLoader } from "./loader-bar.style";
+import { filterStyledSystemMarginProps } from "../../style/utils";
+
+const marginPropTypes = filterStyledSystemMarginProps(
+  styledSystemPropTypes.space
+);
+
+const LoaderBar = ({ size, ...rest }) => {
+  return (
+    <StyledLoader {...rest} {...tagComponent("loader-bar", rest)}>
+      <StyledLoaderBar size={size}>
+        <InnerBar size={size} />
+      </StyledLoaderBar>
+    </StyledLoader>
+  );
+};
+
+LoaderBar.defaultProps = {
```

Comment:
You should use es6 defaults rather than `.defaultProps` as it's going to be deprecated on functional components by React: https://github.com/yannickcr/eslint-plugin-react/issues/2396

So `const LoaderBar = ({ size = "medium", ...rest }) => {` on line 12